pkg/checkers/checker.go

```diff
@@ -53,16 +53,15 @@ type CommandChecker struct {
 }
 
 // RebootRequired for CommandChecker runs a command without returning
-// any eventual error. THis should be later refactored to remove the util wrapper
-// and return the errors, instead of logging them here.
+// any eventual error. This should be later refactored to return the errors,
+// instead of logging and fataling them here.
 func (rc CommandChecker) RebootRequired() bool {
-	var cmdline []string
-	if rc.Privileged {
-		cmdline = util.PrivilegedHostCommand(rc.NamespacePid, rc.CheckCommand)
-	} else {
-		cmdline = rc.CheckCommand
-	}
-	cmd := util.NewCommand(cmdline[0], cmdline[1:]...)
+	bufStdout := new(bytes.Buffer)
+	bufStderr := new(bytes.Buffer)
+	cmd := exec.Command(rc.CheckCommand[0], rc.CheckCommand[1:]...)
+	cmd.Stdout = bufStdout
+	cmd.Stderr = bufStderr
+
 	if err := cmd.Run(); err != nil {
 		switch err := err.(type) {
 		case *exec.ExitError:
```

Comment:
We might want to output stderr and stdout in the log.Warnf below so the user gets important output. Previously this was displayed with the io.Pipe in logrus writer.

Reply:
fine for me.

Reply:
Side note, when error happens, most likely the stderr and stdout will be empty. Do we really need this?

Reply:
it totally depends on the command and hiding the error is not helpful for the end user.